src/index.html

```diff
@@ -1,13 +1,39 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="containerCard" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        alt="Apple195434"
+        class="Apple195434">
+      <a
+        href="#appleA1419_195434"
+        class="A1419Link">Моноблок APPLE A1419 iMac 27" Retina 5k
```

Comment:
Не самый лучший стиль кода
```suggestion
        class="A1419Link"
      >
        Моноблок APPLE A1419 iMac 27" Retina 5k
```